pkg/quota_management/organisation.go

```diff
@@ -1,10 +1,25 @@
 package quota_management
 
+import (
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/shared"
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/shared/utils/arrays"
+)
+
+var defaultQuotaForStandard = Quota{
```

Comment:
Does that mean that the default granted quota is for standard instance types but no billing models are supported? what is the meaning of that?